core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3053,6 +3053,25 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_CACHE_IO_TIMEOUT_DURATION =
+      new Builder(Name.WORKER_CACHE_IO_TIMEOUT_DURATION)
+          .setDefaultValue("-1")
+          .setDescription("The timeout duration for worker cache I/O operations ("
+              + "reading/writing). When this property is a positive value,"
+              + "worker cache operations after timing out will fail and fallback to external "
+              + "file system but transparent to applications; "
+              + "when this property is a negative value, this feature is disabled.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.CLIENT)
+          .build();
+  public static final PropertyKey WORKER_CACHE_IO_TIMEOUT_THREADS =
+      new Builder(Name.WORKER_CACHE_IO_TIMEOUT_THREADS)
+          .setDefaultValue("32")
```

Comment:
Is this number sufficient? the default value of block_read_thread_max much bigger than this one

Reply:
Change to 1024 threads max and suggest users setting this value based on block_read/write threads max